app/code/community/Bolt/Boltpay/controllers/ShippingController.php

```diff
@@ -217,11 +217,11 @@ public function prefetchEstimateAction()
 
             // ----------^_^----------- //
             $shippingAddress = array(
-                'city'       => @$shippingAddressOriginal['city'],
-                'region'     => @$shippingAddressOriginal['region'],
+                'city'       => @$shippingAddressOriginal['city'] ? $shippingAddressOriginal['city'] : null,
```

Comment:
@daisy1754  @mccottry  Personally, I also would like to change it to the style `'city'       => isset($shippingAddressOriginal['city']) ? $shippingAddressOriginal['city'] : null,` , it is much readable and error suppression is not a good idea in such an array
```
Error suppression should be avoided if possible as it doesn't just suppress the error that you are trying to stop, but will also suppress errors that you didn't predict would ever occur. This will make debugging a nightmare.
```

also we can not set it to empty string here, cause if so, when we refresh the cart page, the region id would become 0, not empty anymore, that would cause issue.